README.md

```diff
@@ -1,57 +1,61 @@
-# Lock <img align="right" src="https://github.com/1Hive/website/blob/master/website/static/img/bee.png" height="80px" />
+# Time Lock <img align="right" src="https://github.com/1Hive/website/blob/master/website/static/img/bee.png" height="80px" />
 
 [![CircleCI](https://circleci.com/gh/1Hive/lock-app.svg?style=svg)](https://circleci.com/gh/1Hive/lock-app)
 [![Coverage Status](https://coveralls.io/repos/github/1Hive/lock-app/badge.svg?branch=master&service=github)](https://coveralls.io/github/1Hive/lock-app?branch=master&service=github)
 [![Crytic Status](https://crytic.io/api/repositories/bRQyzLe1Qm2ckDlBsGQKLQ/badge.svg?token=4d485d8b-e6c1-4a3b-9f8e-cb3133ee3c4f)](https://crytic.io/1Hive/lock-app)
 
-1Hive's Lock app allows an Aragon organization to require users to lock tokens by sending them to the Lock app for a configurable period of time in order to forward an intent. For example the organization may require users to lock 100 tokens for 1 month before creating a new vote. The user would be able to come back in a month and claim their deposited tokens.
+1Hive's Time Lock app allows an Aragon organization to require users to lock tokens by sending them to the Time Lock app for a configurable period of time in order to forward an intent. For example the organization may require users to lock 100 tokens for 1 month before creating a new vote. The user would be able to come back in a month and claim their locked tokens.
 
 #### 🐲 Project stage: development
 
-The Lock app is still in development and hasn't been published to APM. If you are interested in contributing please see our open [issues](https://github.com/1hive/lock-app/issues).
+The Time Lock app is still in development, a first implementation was published to `time-lock.open.aragonpm.eth`. If you are interested in contributing please see our open [issues](https://github.com/1hive/time-lock-app/issues).
 
 #### 🚨 Security review status: pre-audit
 
 The code in this repo has not been audited.
 
 ## How does it work
 
-The lock app is a [forwarder](https://hack.aragon.org/docs/forwarding-intro). By granting the lock app a permission like `Create Votes` the user will be prompted and required to lock tokens before the user's intent can be forwarded.
+The Time Lock app is a [forwarder](https://hack.aragon.org/docs/forwarding-intro). By granting the Time Lock app a permission like `Create Votes` the user will be prompted and required to lock tokens before the user's intent can be forwarded.
 
-We do not protect the lock function with a role, so anyone is able to make locks and forward actions. We keep track of when deposits are made and by whom so that users are only able to re-claim deposits that they have made after the duration has elapsed.
+We do not protect the lock function with a role, so anyone is able to make locks and forward actions. We keep track of when locks are made and by whom so that users are only able to re-claim locks that they have made after the duration has elapsed.
```

Comment:
This should be removed as we now check with an Oracle if someone can lock tokens:  `We do not protect the lock function with a role, so anyone is able to make locks and forward actions.`